src/App.php

```diff
@@ -30,52 +31,56 @@
 
 class App
 {
-    const VERSION = '0.1';
+    const VERSION = '0.1.1';
 
     private Dispatcher $dispatcher;
     private ServerRequestInterface $request;
     private ContainerInterface $container;
     private LoggerInterface $logger;
     private float $startTime;
+    /** @var mixed[] */
```

Comment:
@ezimuel

Reply:
I'm noticing many projects using less and less doc notation, with more `type` and `strict` usage. In the days of type-hinting, or when there was no type-hinting at all, it was more important. Now the days of `hinting` are gone.

Reply:
I agree with @adamculp, using typying we can avoid additional phpdoc notations. I usually use it only for `throws` or when type is not enough, as for array. I will add the `@throws` notation.

Reply:
In php8 I often abuse with attributes to validate variables. In my opinion a framework must have docs notations. In the last year laravel spent several months to create all doc notations for the community.

Reply:
Is there any documentation indicating why they went that route? I'm also a fan of doc notations, though perhaps not as strict on them everywhere. If they provide value, yes, if they add noise, not so much.

Reply:
Agree with @adamculp, I'm currently using this approach in the code. Since we are requiring PHP 7.4+ we can also use type properties :boom: requiring less phpdoc.